docs/src/use/migrate-to-10.0.0.md

```diff
@@ -19,6 +19,7 @@ The lists below are ordered roughly by the number of users each change is expect
 - [`eslint:recommended` has been updated](#eslint-recommended)
 - [New configuration file lookup algorithm](#config-lookup-from-file)
 - [Old config format no longer supported](#remove-eslintrc)
+- [JSX references are now tracked](#jsx-reference-tracking)
```

Comment:
I think we should add this to `### Breaking changes for plugin developers` as well, because it can affect custom rules.